test/gpdb/scripts/test_pitr_gpdb6.sh

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env bash
+set -exo pipefail
+
+PGBACKREST_TEST_DIR=/home/gpadmin/test_pgbackrest
+
+# Starting up demo cluster
+source "/usr/local/greenplum-db-devel/greenplum_path.sh"
+pushd gpdb_src/gpAux/gpdemo
+make create-demo-cluster WITH_MIRRORS=true
+source gpdemo-env.sh
+popd
+
+# Creating backup and log directories for pgbackrest
+TEST_NAME=$(basename "${0%.sh}")
+mkdir -p "$PGBACKREST_TEST_DIR/logs/$TEST_NAME"
+mkdir -p "$PGBACKREST_TEST_DIR/$TEST_NAME"
+DATADIR="${MASTER_DATA_DIRECTORY%*/*/*}"
+MASTER=${DATADIR}/qddir/demoDataDir-1
+PRIMARY1=${DATADIR}/dbfast1/demoDataDir0
+PRIMARY2=${DATADIR}/dbfast2/demoDataDir1
+PRIMARY3=${DATADIR}/dbfast3/demoDataDir2
+MIRROR1=${DATADIR}/dbfast_mirror1/demoDataDir0
+MIRROR2=${DATADIR}/dbfast_mirror2/demoDataDir1
+MIRROR3=${DATADIR}/dbfast_mirror3/demoDataDir2
+
+# Filling the pgbackrest.conf configuration file
+cat <<EOF > /etc/pgbackrest.conf
+[seg-1]
+pg1-path=$MASTER
+pg1-port=$PGPORT
+
+[seg0]
+pg1-path=$PRIMARY1
+pg1-port=$((PGPORT+2))
+
+[seg1]
+pg1-path=$PRIMARY2
+pg1-port=$((PGPORT+3))
+
+[seg2]
+pg1-path=$PRIMARY3
+pg1-port=$((PGPORT+4))
+
+[global]
+repo1-path=$PGBACKREST_TEST_DIR/$TEST_NAME
+log-path=$PGBACKREST_TEST_DIR/logs/$TEST_NAME
+start-fast=y
+fork=GPDB
+EOF
+
+# Initializing pgbackrest for GPDB
+for i in -1 0 1 2
+do 
+    PGOPTIONS="-c gp_session_role=utility" pgbackrest --stanza=seg$i stanza-create
+done
+
+# Configuring WAL archiving command
+gpconfig -c archive_mode -v on
+
+gpconfig -c archive_command -v "'PGOPTIONS=\"-c gp_session_role=utility\" \
+pgbackrest --stanza=seg%c archive-push %p'" --skipvalidation
+
+gpstop -ar
+
+# pgbackrest health check
+for i in -1 0 1 2
+do 
+    PGOPTIONS="-c gp_session_role=utility" pgbackrest --stanza=seg$i check
+done
+
+# The test scenario starts here
+# Creating initial dataset
+export PGDATABASE=gpdb_pitr_database
+createdb
```

Comment:
Maybe, it is better to set name for database, otherwise it will be username. For the next execution of `psql` you will need to add param "-d <DATABASE>".

Reply:
```
export PGDATABASE=gpdb_pitr_database
createdb
```
createdb doesn't create a database named gpdb_pitr_database, does it?